### DISCUSSION TITLE
Does this work with SDL 1.2.7 and OpenGL?

### DISCUSSION THREAD

**Discussion by EGAMatsu** [rust9x/rust#30] [Q&A] [answered]
I ask this because I have interest in moving a engine I am working on from C++ to Rust, for speed and memory optimization.

BUT, I 100% NEED SDL 1.2.7 and OpenGL Support.

**Comment by seritools** [comment, marked as answer]
Rust has regular FFI support, so as long as you use or create bindings for SDL and OpenGL for versions that support the Windows versions you want themselves, it should work just fine.

Actually I know for a fact that OpenGL works to some degree as fellow rust9x user `jmi2k` got OpenGL running on 9x.
